hugo/content/protocol/_index.md

```diff
@@ -149,8 +149,7 @@ The commands look like this:
 
 | *Name*           | *Size (bytes)* | *Comment*                                |
 |------------------|----------------|------------------------------------------|
-| Header           | 1              | Framing protocol header including length |
-|                  |                | of the rest of the frame.                |
+| Header           | 1              | Framing protocol header including length of the rest of the frame. |
```

Comment:
Why the change of line break here?

Reply:
Because it is shown as a separate row on the [website](https://dev.tillitis.se/protocol/#firmware-protocol).